If  --add-opens=java.base/java.net=ALL-UNNAMED added to native-maven-plugin's buildArgs, Exception  will be found in Dubbo ExtensionLoader

<dubbo.version>3.3.0-beta.1</dubbo.version>

native-image plugin setting is:

```
<plugin>
                        <groupId>org.graalvm.buildtools</groupId>
                        <artifactId>native-maven-plugin</artifactId>
                        <version>${native-maven-plugin.version}</version>
                        <configuration>
                            <classesDirectory>${project.build.outputDirectory}</classesDirectory>
                            <metadataRepository>
                                <enabled>true</enabled>
<!--                                <url>https://github.com/oracle/graalvm-reachability-metadata/releases/download/0.3.5/graalvm-reachability-metadata-0.3.5.zip</url>-->
<!--                                <version>0.3.5</version>-->
                            </metadataRepository>
                            <requiredVersion>22.3</requiredVersion>
                            <buildArgs>
                               --add-opens=java.base/java.net=ALL-UNNAMED
                            </buildArgs>
                        </configuration>
```


```
10:05:54.612 logback [main] INFO  o.a.d.s.b.c.e.WelcomeLogoApplicationListener -  [DUBBO] 

 :: Dubbo (v3.3.0-beta.1) : https://github.com/apache/dubbo
 :: Discuss group : dev@dubbo.apache.org
, dubbo version: 3.3.0-beta.1, current host: 172.17.0.1

  .   ____          _            __ _ _
 /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  '  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::                (v3.0.9)

10:05:54.625 logback [main] INFO  com.xxx.sail.sink.SinkApplication - Starting AOT-processed SinkApplication using Java 17.0.5 with PID 22863 (/home/sail/app/sail-sink started by root in /home/sail/app)
10:05:54.625 logback [main] INFO  com.xxx.sail.sink.SinkApplication - The following 1 profile is active: "prod"
10:05:54.629 logback [main] INFO  o.a.dubbo.rpc.model.FrameworkModel -  [DUBBO] Dubbo Framework[1] is created, dubbo version: 3.3.0-beta.1, current host: 172.17.0.1
10:05:54.630 logback [main] INFO  o.a.d.c.r.GlobalResourcesRepository -  [DUBBO] Creating global shared handler ..., dubbo version: 3.3.0-beta.1, current host: 172.17.0.1
10:05:54.631 logback [main] ERROR o.a.d.c.extension.ExtensionLoader -  [DUBBO] Exception occurred when loading extension class (interface: interface org.apache.dubbo.common.extension.ExtensionInjector, class file: resource:/META-INF/dubbo/internal/org.apache.dubbo.common.extension.ExtensionInjector#2b374c2d-2024-4746-8c28-da57e73a874d) in resource:/META-INF/dubbo/internal/org.apache.dubbo.common.extension.ExtensionInjector#2b374c2d-2024-4746-8c28-da57e73a874d, dubbo version: 3.3.0-beta.1, current host: 172.17.0.1, error code: 0-15. This may be caused by , go to https://dubbo.apache.org/faq/0/15 to find instructions. 
java.lang.IllegalArgumentException: URL anchor '#2b374c2d-2024-4746-8c28-da57e73a874d' not allowed in resource URL
	at org.graalvm.nativeimage.builder/com.oracle.svm.core.jdk.resources.ResourceURLConnection.connect(ResourceURLConnection.java:70)
	at org.graalvm.nativeimage.builder/com.oracle.svm.core.jdk.resources.ResourceURLConnection.getInputStream(ResourceURLConnection.java:89)
	at java.base@17.0.5/java.net.URL.openStream(URL.java:1161)
	at org.apache.dubbo.common.extension.ExtensionLoader.lambda$getResourceContent$7(ExtensionLoader.java:1169)
	at java.base@17.0.5/java.util.concurrent.ConcurrentHashMap.computeIfAbsent(ConcurrentHashMap.java:1708)
	at org.apache.dubbo.common.extension.ExtensionLoader.getResourceContent(ExtensionLoader.java:1165)
	at org.apache.dubbo.common.extension.ExtensionLoader.loadResource(ExtensionLoader.java:1120)
	at org.apache.dubbo.common.extension.ExtensionLoader.loadFromClass(ExtensionLoader.java:1109)
	at org.apache.dubbo.common.extension.ExtensionLoader.lambda$loadDirectoryInternal$6(ExtensionLoader.java:1090)
	at java.base@17.0.5/java.util.LinkedHashMap.forEach(LinkedHashMap.java:721)
	at java.base@17.0.5/java.util.Collections$UnmodifiableMap.forEach(Collections.java:1553)
	at org.apache.dubbo.common.extension.ExtensionLoader.loadDirectoryInternal(ExtensionLoader.java:1089)
	at org.apache.dubbo.common.extension.ExtensionLoader.loadDirectory(ExtensionLoader.java:1006)
	at org.apache.dubbo.common.extension.ExtensionLoader.loadExtensionClasses(ExtensionLoader.java:993)
	at org.apache.dubbo.common.extension.ExtensionLoader.getExtensionClasses(ExtensionLoader.java:966)
	at org.apache.dubbo.common.extension.ExtensionLoader.getAdaptiveExtensionClass(ExtensionLoader.java:1404)
	at org.apache.dubbo.common.extension.ExtensionLoader.createAdaptiveExtension(ExtensionLoader.java:1391)
	at org.apache.dubbo.common.extension.ExtensionLoader.getAdaptiveExtension(ExtensionLoader.java:745)
	at org.apache.dubbo.common.extension.ExtensionLoader.<init>(ExtensionLoader.java:219)
	at org.apache.dubbo.common.extension.ExtensionDirector.createExtensionLoader0(ExtensionDirector.java:123)
	at org.apache.dubbo.common.extension.ExtensionDirector.createExtensionLoader(ExtensionDirector.java:114)
	at org.apache.dubbo.common.extension.ExtensionDirector.getExtensionLoader(ExtensionDirector.java:104)
	at org.apache.dubbo.common.extension.ExtensionAccessor.getExtensionLoader(ExtensionAccessor.java:27)
	at org.apache.dubbo.metadata.definition.TypeDefinitionBuilder.initBuilders(TypeDefinitionBuilder.java:43)
	at org.apache.dubbo.rpc.model.FrameworkModel.<init>(FrameworkModel.java:90)
	at org.apache.dubbo.rpc.model.FrameworkModel.defaultModel(FrameworkModel.java:181)
	at org.apache.dubbo.config.spring.context.DubboSpringInitializer.customize(DubboSpringInitializer.java:198)
	at org.apache.dubbo.config.spring.context.DubboSpringInitializer.initContext(DubboSpringInitializer.java:108)
	at org.apache.dubbo.config.spring.context.DubboSpringInitializer.initialize(DubboSpringInitializer.java:65)
	at org.apache.dubbo.config.spring.context.DubboContextPostProcessor.postProcessBeanDefinitionRegistry(DubboContextPostProcessor.java:72)
	at org.springframework.context.support.PostProcessorRegistrationDelegate.invokeBeanDefinitionRegistryPostProcessors(PostProcessorRegistrationDelegate.java:344)
	at org.springframework.context.support.PostProcessorRegistrationDelegate.invokeBeanFactoryPostProcessors(PostProcessorRegistrationDelegate.java:145)
	at org.springframework.context.support.AbstractApplicationContext.invokeBeanFactoryPostProcessors(AbstractApplicationContext.java:771)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:589)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:146)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:732)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:434)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:310)
	at org.springframework.boot.builder.SpringApplicationBuilder.run(SpringApplicationBuilder.java:150)
	at com.xxx.sail.sink.SinkApplication.main(SinkApplication.java:143)
10:05:54.631 logback [main] ERROR o.a.d.c.extension.ExtensionLoader -  [DUBBO] Exception occurred when loading extension class (interface: interface org.apache.dubbo.common.extension.ExtensionInjector, class file: resource:/META-INF/dubbo/internal/org.apache.dubbo.common.extension.ExtensionInjector#068847e4-062c-481e-b9cf-bac03c2e2f8f) in resource:/META-INF/dubbo/internal/org.apache.dubbo.common.extension.ExtensionInjector#068847e4-062c-481e-b9cf-bac03c2e2f8f, dubbo version: 3.3.0-beta.1, current host: 172.17.0.1, error code: 0-15. This may be caused by , go to https://dubbo.apache.org/faq/0/15 to find instructions. 
java.lang.IllegalArgumentException: URL anchor '#068847e4-062c-481e-b9cf-bac03c2e2f8f' not allowed in resource URL
	at org.graalvm.nativeimage.builder/com.oracle.svm.core.jdk.resources.ResourceURLConnection.connect(ResourceURLConnection.java:70)
	at org.graalvm.nativeimage.builder/com.oracle.svm.core.jdk.resources.ResourceURLConnection.getInputStream(ResourceURLConnection.java:89)
	at java.base@17.0.5/java.net.URL.openStream(URL.java:1161)
	at org.apache.dubbo.common.extension.ExtensionLoader.lambda$getResourceContent$7(ExtensionLoader.java:1169)
	at java.base@17.0.5/java.util.concurrent.ConcurrentHashMap.computeIfAbsent(ConcurrentHashMap.java:1708)
	at org.apache.dubbo.common.extension.ExtensionLoader.getResourceContent(ExtensionLoader.java:1165)
	at org.apache.dubbo.common.extension.ExtensionLoader.loadResource(ExtensionLoader.java:1120)
	at org.apache.dubbo.common.extension.ExtensionLoader.loadFromClass(ExtensionLoader.java:1109)
	at org.apache.dubbo.common.extension.ExtensionLoader.lambda$loadDirectoryInternal$6(ExtensionLoader.java:1090)
	at java.base@17.0.5/java.util.LinkedHashMap.forEach(LinkedHashMap.java:721)
	at java.base@17.0.5/java.util.Collections$UnmodifiableMap.forEach(Collections.java:1553)
	at org.apache.dubbo.common.extension.ExtensionLoader.loadDirectoryInternal(ExtensionLoader.java:1089)
	at org.apache.dubbo.common.extension.ExtensionLoader.loadDirectory(ExtensionLoader.java:1006)
	at org.apache.dubbo.common.extension.ExtensionLoader.loadExtensionClasses(ExtensionLoader.java:993)
	at org.apache.dubbo.common.extension.ExtensionLoader.getExtensionClasses(ExtensionLoader.java:966)
	at org.apache.dubbo.common.extension.ExtensionLoader.getAdaptiveExtensionClass(ExtensionLoader.java:1404)
	at org.apache.dubbo.common.extension.ExtensionLoader.createAdaptiveExtension(ExtensionLoader.java:1391)
	at org.apache.dubbo.common.extension.ExtensionLoader.getAdaptiveExtension(ExtensionLoader.java:745)
	at org.apache.dubbo.common.extension.ExtensionLoader.<init>(ExtensionLoader.java:219)
	at org.apache.dubbo.common.extension.ExtensionDirector.createExtensionLoader0(ExtensionDirector.java:123)
	at org.apache.dubbo.common.extension.ExtensionDirector.createExtensionLoader(ExtensionDirector.java:114)
	at org.apache.dubbo.common.extension.ExtensionDirector.getExtensionLoader(ExtensionDirector.java:104)
	at org.apache.dubbo.common.extension.ExtensionAccessor.getExtensionLoader(ExtensionAccessor.java:27)
	at org.apache.dubbo.metadata.definition.TypeDefinitionBuilder.initBuilders(TypeDefinitionBuilder.java:43)
	at org.apache.dubbo.rpc.model.FrameworkModel.<init>(FrameworkModel.java:90)
	at org.apache.dubbo.rpc.model.FrameworkModel.defaultModel(FrameworkModel.java:181)
	at org.apache.dubbo.config.spring.context.DubboSpringInitializer.customize(DubboSpringInitializer.java:198)
	at org.apache.dubbo.config.spring.context.DubboSpringInitializer.initContext(DubboSpringInitializer.java:108)
	at org.apache.dubbo.config.spring.context.DubboSpringInitializer.initialize(DubboSpringInitializer.java:65)
	at org.apache.dubbo.config.spring.context.DubboContextPostProcessor.postProcessBeanDefinitionRegistry(DubboContextPostProcessor.java:72)
	at org.springframework.context.support.PostProcessorRegistrationDelegate.invokeBeanDefinitionRegistryPostProcessors(PostProcessorRegistrationDelegate.java:344)
	at org.springframework.context.support.PostProcessorRegistrationDelegate.invokeBeanFactoryPostProcessors(PostProcessorRegistrationDelegate.java:145)
	at org.springframework.context.support.AbstractApplicationContext.invokeBeanFactoryPostProcessors(AbstractApplicationContext.java:771)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:589)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:146)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:732)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:434)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:310)
	at org.springframework.boot.builder.SpringApplicationBuilder.run(SpringApplicationBuilder.java:150)
	at com.xxx.sail.sink.SinkApplication.main(SinkApplication.java:143)
10:05:54.632 logback [main] ERROR o.a.d.c.extension.ExtensionLoader -  [DUBBO] Exception occurred when loading extension class (interface: interface org.apache.dubbo.common.extension.ExtensionInjector, class file: resource:/META-INF/dubbo/internal/org.apache.dubbo.common.extension.ExtensionInjector#7fb37ae6-3667-4ad7-8d90-e5dfc54f8290) in resource:/META-INF/dubbo/internal/org.apache.dubbo.common.extension.ExtensionInjector#7fb37ae6-3667-4ad7-8d90-e5dfc54f8290, dubbo version: 3.3.0-beta.1, current host: 172.17.0.1, error code: 0-15. This may be caused by , go to https://dubbo.apache.org/faq/0/15 to find instructions. 
java.lang.IllegalArgumentException: URL anchor '#7fb37ae6-3667-4ad7-8d90-e5dfc54f8290' not allowed in resource URL
	at org.graalvm.nativeimage.builder/com.oracle.svm.core.jdk.resources.ResourceURLConnection.connect(ResourceURLConnection.java:70)
	at org.graalvm.nativeimage.builder/com.oracle.svm.core.jdk.resources.ResourceURLConnection.getInputStream(ResourceURLConnection.java:89)
	at java.base@17.0.5/java.net.URL.openStream(URL.java:1161)
	at org.apache.dubbo.common.extension.ExtensionLoader.lambda$getResourceContent$7(ExtensionLoader.java:1169)
	at java.base@17.0.5/java.util.concurrent.ConcurrentHashMap.computeIfAbsent(ConcurrentHashMap.java:1708)
	at org.apache.dubbo.common.extension.ExtensionLoader.getResourceContent(ExtensionLoader.java:1165)
	at org.apache.dubbo.common.extension.ExtensionLoader.loadResource(ExtensionLoader.java:1120)
	at org.apache.dubbo.common.extension.ExtensionLoader.loadFromClass(ExtensionLoader.java:1109)
	at org.apache.dubbo.common.extension.ExtensionLoader.lambda$loadDirectoryInternal$6(ExtensionLoader.java:1090)
	at java.base@17.0.5/java.util.LinkedHashMap.forEach(LinkedHashMap.java:721)
	at java.base@17.0.5/java.util.Collections$UnmodifiableMap.forEach(Collections.java:1553)
	at org.apache.dubbo.common.extension.ExtensionLoader.loadDirectoryInternal(ExtensionLoader.java:1089)
	at org.apache.dubbo.common.extension.ExtensionLoader.loadDirectory(ExtensionLoader.java:1006)
	at org.apache.dubbo.common.extension.ExtensionLoader.loadExtensionClasses(ExtensionLoader.java:993)
	at org.apache.dubbo.common.extension.ExtensionLoader.getExtensionClasses(ExtensionLoader.java:966)
	at org.apache.dubbo.common.extension.ExtensionLoader.getAdaptiveExtensionClass(ExtensionLoader.java:1404)
	at org.apache.dubbo.common.extension.ExtensionLoader.createAdaptiveExtension(ExtensionLoader.java:1391)
	at org.apache.dubbo.common.extension.ExtensionLoader.getAdaptiveExtension(ExtensionLoader.java:745)
	at org.apache.dubbo.common.extension.ExtensionLoader.<init>(ExtensionLoader.java:219)
	at org.apache.dubbo.common.extension.ExtensionDirector.createExtensionLoader0(ExtensionDirector.java:123)
	at org.apache.dubbo.common.extension.ExtensionDirector.createExtensionLoader(ExtensionDirector.java:114)
	at org.apache.dubbo.common.extension.ExtensionDirector.getExtensionLoader(ExtensionDirector.java:104)
	at org.apache.dubbo.common.extension.ExtensionAccessor.getExtensionLoader(ExtensionAccessor.java:27)
	at org.apache.dubbo.metadata.definition.TypeDefinitionBuilder.initBuilders(TypeDefinitionBuilder.java:43)
	at org.apache.dubbo.rpc.model.FrameworkModel.<init>(FrameworkModel.java:90)
	at org.apache.dubbo.rpc.model.FrameworkModel.defaultModel(FrameworkModel.java:181)
	at org.apache.dubbo.config.spring.context.DubboSpringInitializer.customize(DubboSpringInitializer.java:198)
	at org.apache.dubbo.config.spring.context.DubboSpringInitializer.initContext(DubboSpringInitializer.java:108)
	at org.apache.dubbo.config.spring.context.DubboSpringInitializer.initialize(DubboSpringInitializer.java:65)
	at org.apache.dubbo.config.spring.context.DubboContextPostProcessor.postProcessBeanDefinitionRegistry(DubboContextPostProcessor.java:72)
	at org.springframework.context.support.PostProcessorRegistrationDelegate.invokeBeanDefinitionRegistryPostProcessors(PostProcessorRegistrationDelegate.java:344)
	at org.springframework.context.support.PostProcessorRegistrationDelegate.invokeBeanFactoryPostProcessors(PostProcessorRegistrationDelegate.java:145)
	at org.springframework.context.support.AbstractApplicationContext.invokeBeanFactoryPostProcessors(AbstractApplicationContext.java:771)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:589)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:146)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:732)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:434)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:310)
	at org.springframework.boot.builder.SpringApplicationBuilder.run(SpringApplicationBuilder.java:150)
	at com.xxx.sail.sink.SinkApplication.main(SinkApplication.java:143)
10:05:54.635 logback [main] WARN  o.a.d.c.s.r.ReferenceBeanManager -  [DUBBO] Early initialize reference bean before DubboConfigBeanInitializer, the BeanPostProcessor has not been loaded at this time, which may cause abnormalities in some components (such as seata): dubboSinkMonitorService = ReferenceBean:com.xxx.sail.foundation.api.service.DubboSinkMonitorService(), dubbo version: 3.3.0-beta.1, current host: 172.17.0.1, error code: 5-39. This may be caused by , go to https://dubbo.apache.org/faq/5/39 to find instructions. 
10:05:54.635 logback [main] WARN  o.a.d.c.s.r.ReferenceBeanManager -  [DUBBO] Early initialize reference bean before DubboConfigBeanInitializer, the BeanPostProcessor has not been loaded at this time, which may cause abnormalities in some components (such as seata): dubboSettingService = ReferenceBean:com.xxx.sail.foundation.api.service.DubboSettingService(), dubbo version: 3.3.0-beta.1, current host: 172.17.0.1, error code: 5-39. This may be caused by , go to https://dubbo.apache.org/faq/5/39 to find instructions. 
10:05:54.635 logback [main] WARN  o.a.d.c.s.r.ReferenceBeanManager -  [DUBBO] Early initialize reference bean before DubboConfigBeanInitializer, the BeanPostProcessor has not been loaded at this time, which may cause abnormalities in some components (such as seata): dubboSinkService = ReferenceBean:com.xxx.sail.foundation.api.service.DubboSinkService(), dubbo version: 3.3.0-beta.1, current host: 172.17.0.1, error code: 5-39. This may be caused by , go to https://dubbo.apache.org/faq/5/39 to find instructions. 
10:05:54.635 logback [main] WARN  o.a.d.c.s.r.ReferenceBeanManager -  [DUBBO] Early initialize reference bean before DubboConfigBeanInitializer, the BeanPostProcessor has not been loaded at this time, which may cause abnormalities in some components (such as seata): dubboAuditCacheService = ReferenceBean:com.xxx.sail.foundation.api.service.DubboAuditCacheService(), dubbo version: 3.3.0-beta.1, current host: 172.17.0.1, error code: 5-39. This may be caused by , go to https://dubbo.apache.org/faq/5/39 to find instructions. 
10:05:54.636 logback [main] WARN  o.a.d.c.s.r.ReferenceBeanManager -  [DUBBO] Early initialize reference bean before DubboConfigBeanInitializer, the BeanPostProcessor has not been loaded at this time, which may cause abnormalities in some components (such as seata): dubboAlarmLogService = ReferenceBean:com.xxx.sail.foundation.api.service.DubboAlarmLogService(), dubbo version: 3.3.0-beta.1, current host: 172.17.0.1, error code: 5-39. This may be caused by , go to https://dubbo.apache.org/faq/5/39 to find instructions. 
10:05:54.636 logback [main] WARN  o.a.d.c.s.r.ReferenceBeanManager -  [DUBBO] Early initialize reference bean before DubboConfigBeanInitializer, the BeanPostProcessor has not been loaded at this time, which may cause abnormalities in some components (such as seata): dubboRepricingService = ReferenceBean:com.xxx.sail.foundation.api.service.DubboRepricingService(), dubbo version: 3.3.0-beta.1, current host: 172.17.0.1, error code: 5-39. This may be caused by , go to https://dubbo.apache.org/faq/5/39 to find instructions. 
10:05:54.636 logback [main] WARN  o.a.d.c.s.r.ReferenceBeanManager -  [DUBBO] Early initialize reference bean before DubboConfigBeanInitializer, the BeanPostProcessor has not been loaded at this time, which may cause abnormalities in some components (such as seata): dubboSinkPriorityService = ReferenceBean:com.xxx.sail.foundation.api.service.DubboSinkPriorityService(), dubbo version: 3.3.0-beta.1, current host: 172.17.0.1, error code: 5-39. This may be caused by , go to https://dubbo.apache.org/faq/5/39 to find instructions. 
10:05:54.636 logback [main] WARN  o.a.d.c.s.r.ReferenceBeanManager -  [DUBBO] Early initialize reference bean before DubboConfigBeanInitializer, the BeanPostProcessor has not been loaded at this time, which may cause abnormalities in some components (such as seata): dubboTemplateService = ReferenceBean:com.xxx.sail.foundation.api.service.DubboTemplateService(), dubbo version: 3.3.0-beta.1, current host: 172.17.0.1, error code: 5-39. This may be caused by , go to https://dubbo.apache.org/faq/5/39 to find instructions. 
10:05:54.636 logback [main] WARN  o.a.d.c.s.r.ReferenceBeanManager -  [DUBBO] Early initialize reference bean before DubboConfigBeanInitializer, the BeanPostProcessor has not been loaded at this time, which may cause abnormalities in some components (such as seata): dubboBlackGroupService = ReferenceBean:com.xxx.sail.foundation.api.service.DubboBlackGroupService(), dubbo version: 3.3.0-beta.1, current host: 172.17.0.1, error code: 5-39. This may be caused by , go to https://dubbo.apache.org/faq/5/39 to find instructions. 
10:05:54.636 logback [main] WARN  o.a.d.c.s.r.ReferenceBeanManager -  [DUBBO] Early initialize reference bean before DubboConfigBeanInitializer, the BeanPostProcessor has not been loaded at this time, which may cause abnormalities in some components (such as seata): dubboBlackPoolService = ReferenceBean:com.xxx.sail.foundation.api.service.DubboBlackPoolService(), dubbo version: 3.3.0-beta.1, current host: 172.17.0.1, error code: 5-39. This may be caused by , go to https://dubbo.apache.org/faq/5/39 to find instructions. 
10:05:54.636 logback [main] WARN  o.a.d.c.s.r.ReferenceBeanManager -  [DUBBO] Early initialize reference bean before DubboConfigBeanInitializer, the BeanPostProcessor has not been loaded at this time, which may cause abnormalities in some components (such as seata): dubboBlackPoolMobileService = ReferenceBean:com.xxx.sail.foundation.api.service.DubboBlackPoolMobileService(), dubbo version: 3.3.0-beta.1, current host: 172.17.0.1, error code: 5-39. This may be caused by , go to https://dubbo.apache.org/faq/5/39 to find instructions. 
10:05:54.636 logback [main] WARN  o.a.d.c.s.r.ReferenceBeanManager -  [DUBBO] Early initialize reference bean before DubboConfigBeanInitializer, the BeanPostProcessor has not been loaded at this time, which may cause abnormalities in some components (such as seata): dubboSinkGroupService = ReferenceBean:com.xxx.sail.foundation.api.service.DubboSinkGroupService(), dubbo version: 3.3.0-beta.1, current host: 172.17.0.1, error code: 5-39. This may be caused by , go to https://dubbo.apache.org/faq/5/39 to find instructions. 
10:05:54.637 logback [main] ERROR o.s.boot.SpringApplication - Application run failed
java.lang.IllegalStateException: Failed to create adaptive instance: java.lang.IllegalStateException: Can't create adaptive extension interface org.apache.dubbo.common.extension.ExtensionInjector, cause: No adaptive method exist on extension org.apache.dubbo.common.extension.ExtensionInjector, refuse to create the adaptive class!
	at org.apache.dubbo.common.extension.ExtensionLoader.getAdaptiveExtension(ExtensionLoader.java:750)
	at org.apache.dubbo.common.extension.ExtensionLoader.<init>(ExtensionLoader.java:219)
	at org.apache.dubbo.common.extension.ExtensionDirector.createExtensionLoader0(ExtensionDirector.java:123)
	at org.apache.dubbo.common.extension.ExtensionDirector.createExtensionLoader(ExtensionDirector.java:114)
	at org.apache.dubbo.common.extension.ExtensionDirector.getExtensionLoader(ExtensionDirector.java:104)
	at org.apache.dubbo.common.extension.ExtensionAccessor.getExtensionLoader(ExtensionAccessor.java:27)
	at org.apache.dubbo.metadata.definition.TypeDefinitionBuilder.initBuilders(TypeDefinitionBuilder.java:43)
	at org.apache.dubbo.rpc.model.FrameworkModel.<init>(FrameworkModel.java:90)
	at org.apache.dubbo.rpc.model.FrameworkModel.defaultModel(FrameworkModel.java:181)
	at org.apache.dubbo.config.spring.context.DubboSpringInitializer.customize(DubboSpringInitializer.java:198)
	at org.apache.dubbo.config.spring.context.DubboSpringInitializer.initContext(DubboSpringInitializer.java:108)
	at org.apache.dubbo.config.spring.context.DubboSpringInitializer.initialize(DubboSpringInitializer.java:65)
	at org.apache.dubbo.config.spring.context.DubboContextPostProcessor.postProcessBeanDefinitionRegistry(DubboContextPostProcessor.java:72)
	at org.springframework.context.support.PostProcessorRegistrationDelegate.invokeBeanDefinitionRegistryPostProcessors(PostProcessorRegistrationDelegate.java:344)
	at org.springframework.context.support.PostProcessorRegistrationDelegate.invokeBeanFactoryPostProcessors(PostProcessorRegistrationDelegate.java:145)
	at org.springframework.context.support.AbstractApplicationContext.invokeBeanFactoryPostProcessors(AbstractApplicationContext.java:771)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:589)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:146)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:732)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:434)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:310)
	at org.springframework.boot.builder.SpringApplicationBuilder.run(SpringApplicationBuilder.java:150)
	at com.xxx.sail.sink.SinkApplication.main(SinkApplication.java:143)
Caused by: java.lang.IllegalStateException: Can't create adaptive extension interface org.apache.dubbo.common.extension.ExtensionInjector, cause: No adaptive method exist on extension org.apache.dubbo.common.extension.ExtensionInjector, refuse to create the adaptive class!
	at org.apache.dubbo.common.extension.ExtensionLoader.createAdaptiveExtension(ExtensionLoader.java:1399)
	at org.apache.dubbo.common.extension.ExtensionLoader.getAdaptiveExtension(ExtensionLoader.java:745)
	... 22 common frames omitted
Caused by: java.lang.IllegalStateException: No adaptive method exist on extension org.apache.dubbo.common.extension.ExtensionInjector, refuse to create the adaptive class!
	at org.apache.dubbo.common.extension.AdaptiveClassCodeGenerator.generate(AdaptiveClassCodeGenerator.java:105)
	at org.apache.dubbo.common.extension.AdaptiveClassCodeGenerator.generate(AdaptiveClassCodeGenerator.java:95)
	at org.apache.dubbo.common.extension.ExtensionLoader.createAdaptiveExtensionClass(ExtensionLoader.java:1421)
	at org.apache.dubbo.common.extension.ExtensionLoader.getAdaptiveExtensionClass(ExtensionLoader.java:1408)
	at org.apache.dubbo.common.extension.ExtensionLoader.createAdaptiveExtension(ExtensionLoader.java:1391)
	... 23 common frames omitted
```

--- a/dubbo-common/src/main/java/org/apache/dubbo/common/utils/ClassLoaderResourceLoader.java
+++ b/dubbo-common/src/main/java/org/apache/dubbo/common/utils/ClassLoaderResourceLoader.java
@@ -16,14 +16,12 @@
  */
 package org.apache.dubbo.common.utils;
 
-import org.apache.dubbo.common.aot.NativeDetector;
 import org.apache.dubbo.common.logger.ErrorTypeAwareLogger;
 import org.apache.dubbo.common.logger.LoggerFactory;
 import org.apache.dubbo.common.resource.GlobalResourcesRepository;
 
 import java.io.IOException;
 import java.lang.ref.SoftReference;
-import java.lang.reflect.Field;
 import java.net.URL;
 import java.util.Collection;
 import java.util.Collections;
@@ -32,7 +30,6 @@ import java.util.LinkedHashMap;
 import java.util.LinkedHashSet;
 import java.util.Map;
 import java.util.Set;
-import java.util.UUID;
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.CountDownLatch;
 
@@ -81,15 +78,9 @@ public class ClassLoaderResourceLoader {
             Enumeration<URL> urls;
             try {
                 urls = currentClassLoader.getResources(fileName);
-                boolean isNative = NativeDetector.inNativeImage();
                 if (urls != null) {
                     while (urls.hasMoreElements()) {
                         URL url = urls.nextElement();
-                        if (isNative) {
-                            // In native mode, the address of each URL is the same instead of different paths, so it is
-                            // necessary to set the ref to make it different
-                            setRef(url);
-                        }
                         set.add(url);
                     }
                 }
@@ -110,15 +101,6 @@ public class ClassLoaderResourceLoader {
     public static void destroy() {
         synchronized (ClassLoaderResourceLoader.class) {
             classLoaderResourcesCache = null;
-        }
-    }
-
-    private static void setRef(URL url) {
-        try {
-            Field field = URL.class.getDeclaredField("ref");
-            field.setAccessible(true);
-            field.set(url, UUID.randomUUID().toString());
-        } catch (Throwable ignore) {
         }
     }
 

